test/clienthellotest.c

```diff
@@ -56,10 +56,26 @@ int main(int argc, char *argv[])
     for (; currtest < TOTAL_NUM_TESTS; currtest++) {
         testresult = 0;
         ctx = SSL_CTX_new(TLS_method());
+
+        /*
+         * This test is testing session tickets for <= TLS1.2. It isn't relevant
```

Comment:
"Testing for session tickets <=- TLS1.2; not relevant for 1.3" might fit into a single line.  If not, never mind.